### ISSUE TITLE
Glossary definition for blockchain is misleading

### ISSUE THREAD

**Issue by billygarrison** [renepickhardt/The-Lightning-Network-Book#16]
The part about someone with 51% of the hashrate being able to change the data is not clear/true. They can change current blocks to censor or double-spend transactions, but they can't go back through the blockchain and change old data using 51%.

**Comment by renepickhardt** [comment]
they can. If I have 51 % or more hashpower I will be able to start from the genesis block and eventually catch up. the protocol says the chain with the highest proof of work will win. Obviously It would take an increadible amount of time. But assume I have 10x more hashpower than the current network it would only take about 1 year to provide as much proof of work as has been provided in the last 10 years. and another tenth of a year to catch up with the 11th year. Therby I could basically change the entire blockchain (even in the sense that I would send all coinbase tx to my account and basically remove all tx that have ever taken place)

**Comment by billygarrison** [comment]
I haven't seen the math, but at 51% I'm pretty sure it would take an extremely long time to undo blocks. Potentially centuries? Especially for blocks that are far back in the blockchain.

Perhaps a simple rewording would be useful so that it doesn't look like they can just change data at will once they have 51%.

**Comment by renepickhardt** [comment]
Yeah as mentioned with 90% it will take a year to redo the second block. So yes I will try to do another less confusing formulation
